test/Test/Data.hs

```diff
@@ -18,32 +18,37 @@ import Hintman.Hint (getAllComments)
 
 
 data Prs = Prs
-    { pr1  :: !PrInfo
-    , pr2  :: !PrInfo
-    , pr3  :: !PrInfo
-    , pr24 :: !PrInfo
-    , pr30 :: !PrInfo
-    , pr36 :: !PrInfo
+    { pr2     :: !PrInfo
+    , pr3     :: !PrInfo
+    , pr4     :: !PrInfo
+    , pr24    :: !PrInfo
+    , pr30    :: !PrInfo
+    , pr36    :: !PrInfo
+    , forkPr1 :: !PrInfo
     }
 
-owner :: Owner
-owner = Owner "kowainik"
+kowainik :: Owner
+kowainik = Owner "kowainik"
 
 repo :: Repo
 repo = Repo "hintman-target"
 
 fetchPrs :: IO Prs
 fetchPrs = usingLoggerT richMessageAction $ do
-    pr1  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
-    pr2  <- makePr (PrNumber 3) (Sha "vrom911-patch-2")
-    pr3  <- makePr (PrNumber 4) (Sha "chshersh/parse-error")
+    pr2  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
```

Comment:
When I was creating tests, I firstly looked at already implemented ones and couldn't get the logic for a pretty long time -- expected comments just didn't match the actual contents of PRs 1/2/3.
Then I discovered this inconsistency :upside_down_face: 
Hopefully, it's OK that I decided to change numbers to the actual ones.

Reply:
I think the consistency of variable names and PR numbers is okay, no need to change back :+1: